resources/test/tox.ini

```diff
@@ -13,4 +13,4 @@ deps =
   selenium
   requests
 
-commands = pytest {posargs} -vv tests
+commands = pytest {posargs} -vv tests/functional/idlharness
```

Comment:
This is local debugging accidentally committed, right?

Reply:
Yah, my bad.